R/backend-dbplyr__duckdb_connection.R

```diff
@@ -302,6 +302,17 @@ sql_translation.duckdb_connection <- function(con) {
         build_sql("DATEDIFF('day', ", !!start, ", " ,!!end, ")")
 
       },
+      date_build = function(year, month = 1L, day = 1L, ..., invalid = NULL) {
+        dbplyr:::check_unsupported_arg(invalid, allow_null = TRUE)
```

Comment:
I'd rather not access a function private to dbplyr here. How do other packages handle this? As a last resort, we could vendor a variant of `check_unsupported_arg()` here.